eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -445,16 +445,6 @@
     <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
   </Match>
 
-  <!-- Incorrect flagging, if the response is null a HttpResponseException should be thrown
-  https://github.com/azure/azure-sdk-for-java/issues/4556 -->
-  <Match>
-    <Or>
-      <Class name="com.azure.security.keyvault.keys.KeyClient"/>
```

Comment:
how was this spotbug issue resolved?

Reply:
This was brought in from merge - will revert.